apps/condo/domains/notification/utils/serverSchema/index.js

```diff
@@ -28,6 +29,7 @@ const {
 } = require('@condo/domains/notification/gql')
 /* AUTOGENERATE MARKER <IMPORT> */
 
+const logger = getLogger('notification/serverSchema')
```

Comment:
I guess, you've added this log record because something was wrong with `sendMessage` mutation.
I've created a search [All GraphQL Mutations](https://app.datadoghq.eu/apm/traces?saved-view-id=76418) in DataDog. It does not displays executions of `sendMessage` mutation. Other mutations are flowing in livestream.